botometerAnalyser/cache.js

```diff
@@ -28,14 +28,25 @@ function getUserByName(screenName) {
 		.value();
 }
 
-function getUserById(id) {
+function getUserById(userId) {
 	return db.get('users')
-		.find({ id })
+		.find({ id: userId })
 		.value();
 }
 
+function getUser({ screenName, userId }) {
+	if (userId) {
+		return getUserById(userId);
+	}
+	if (screenName) {
```

Comment:
As it's never used, I think we should completely remove `getUserByName`, it's bug prone

Reply:
done